its/ruling/src/test/java/org/sonar/python/it/PythonRulingTest.java

```diff
@@ -29,11 +29,13 @@
 import org.junit.Test;
 
 import com.google.common.collect.ImmutableList;
-import com.google.common.io.Files;
+
+
 import com.sonar.orchestrator.Orchestrator;
 import com.sonar.orchestrator.build.SonarRunner;
 import com.sonar.orchestrator.locator.FileLocation;
 
+import com.google.common.io.Files;
```

Comment:
@vilchik-elena I suggest I create a ticket to format all classes in one commit. Should solve the issue permanently